### PR TITLE
Release 0.8.0

### PR DESCRIPTION
**Pull request by ffinfo**
This is the pull request for release 0.8.0

@sndrtj: For review only change from this branch to develop are imported, the rest is already reviewed.

### CODE REVIEW

**Comment by ffinfo** [comment]
continuous-integration/jenkins/pr-merge will fail because master does not yet have a jenkins file. This can be ignored

File: generate-indexes/src/test/scala/nl/lumc/sasc/biopet/pipelines/generateindexes/DownloadGenomesTest.scala [hunk 1]
```diff
@@ -45,7 +45,7 @@ class DownloadGenomesTest extends TestNGSuite with Matchers {
   @Test
   def testNcbiAssembly(): Unit = {
     val pipeline = initPipeline(Map())
-    pipeline.referenceConfig = Map("s1" -> Map("g1" -> Map("ncbi_assembly_id" -> "id")))
+    pipeline.referenceConfig = Map("s1" -> Map("g1" -> Map("ncbi_assembly_report" -> "id")))
```
Comment (sndrtj) [inline generate-indexes/src/test/scala/nl/lumc/sasc/biopet/pipelines/generateindexes/DownloadGenomesTest.scala:48]: This is now called 'report' in here and tools, but still 'assembly id' in the tool extension. To be fully consistent, I think we should change that there as well.

**Review by sndrtj — COMMENTED** [review]
One tiny comment on assembly report. See the commit [here](https://github.com/biopet/biopet/pull/15/commits/97d82bfc4b8822b16118fef25bb1a3fd12bc9817)

File: generate-indexes/src/test/scala/nl/lumc/sasc/biopet/pipelines/generateindexes/DownloadGenomesTest.scala (hunk 1 above)
Comment (ffinfo) [reply generate-indexes/src/test/scala/nl/lumc/sasc/biopet/pipelines/generateindexes/DownloadGenomesTest.scala:48]: agree, commit is coming up soon

**Pull request merged**